api/config/dex/config_request_test.go

```diff
@@ -26,7 +26,7 @@ func completeMSADWithoutOverrides() *dex.ConfigRequest_V1_Msad_Ldap {
 func TestValidate(t *testing.T) {
 	t.Run("Validates when there are no connectors", func(t *testing.T) {
 		cfg := dex.DefaultConfigRequest()
-		assert.Nil(t, cfg.Validate())
+		assert.NoError(t, cfg.Validate())
```

Comment:
drive-by cleanup now that `cfg.Validate()` returns something of interface `error`